lib/utils/app_exceptions.dart

```diff
@@ -0,0 +1,57 @@
+class AppException implements Exception {
+  final _message;
+  final _prefix;
+
+  AppException([this._message, this._prefix]);
+
+  String get message => _message;
+
+  String get pefix => _prefix;
```

Comment:
intentional?

Reply:
We may require the error message, so yes intentional..

Reply:
I mean using the var pefix instead of prefix

Reply:
Ah Shit! Here we go again.. this is a typo.. :|

Reply:
Corrected @tachyons